tests/test_html.py

```diff
@@ -175,6 +175,29 @@ def content_summary():
     )
 
 
+def test_text_to_html_takes_one_string_value_and_returns_markup():
+    assert text_to_html("") == Markup("")
+    assert text_to_html("Hello World") == Markup("Hello World")
+
+
+def test_text_to_html_can_capitalize_first_letter():
+    assert text_to_html("hello world", capitalize_first=True) == "Hello world"
+    assert text_to_html("hello world", capitalize_first=False) == "hello world"
+
+
+def test_text_to_html_can_format_links():
+    assert text_to_html("https://gov.uk", format_links=True) == \
+        Markup('<a href="https://gov.uk" class="app-break-link" rel="external">https://gov.uk</a>')
+    assert text_to_html("https://gov.uk", format_links=False) == "https://gov.uk"
```

Comment:
Similar check here (passing no kwargs) just to guard against link formatting being done automatically somewhere in the future?